core/src/main/java/io/substrait/relation/RelProtoConverter.java

```diff
@@ -345,13 +348,8 @@ private List<ConsistentPartitionWindowRel.WindowRelFunction> toProtoWindowRelFun
                       .mapToObj(i -> args.get(i).accept(aggFuncDef, i, argVisitor))
                       .collect(Collectors.toList());
               var options =
-                  f.options().entrySet().stream()
-                      .map(
-                          o ->
-                              FunctionOption.newBuilder()
-                                  .setName(o.getKey())
-                                  .addAllPreference(o.getValue().values())
-                                  .build())
+                  f.options().stream()
+                      .map(ExpressionProtoConverter::from)
```

Comment:
meta: I think re-using this is good, but also think it's a bit weird that this lives in `ExpressionProtoConverter` and is used in `RelProtoConverter`. I'm not sure what a better place for this is yet, so I think it's fine to keep this here for now.